### PR TITLE
WebHost: reduce server uptime

### PR DESCRIPTION
**Pull request by Berserker66**
WebHost: attempt to improve wording of server resume
WebHost: reduce default room timeout to 1 hour

### CODE REVIEW

File: WebHostLib/templates/hostRoom.html [hunk 1]
```diff
@@ -16,9 +16,9 @@
             This room has a <a href="{{ url_for("getTracker", tracker=room.tracker) }}">Multiworld Tracker</a> enabled.
             <br />
         {% endif %}
-        This room will be closed after {{ room.timeout//60//60 }} hours of inactivity. Should you wish to continue
-        later,
-        you can simply refresh this page and the server will be started again.<br>
+        The server for this room will be closed after {{ room.timeout//60//60 }} hours of inactivity.
```
Comment (black-sliver) [inline WebHostLib/templates/hostRoom.html:19]:
```suggestion
        The server for this room will be suspended after {{ room.timeout//60//60 }} hours of inactivity.
```
As per suggestion in discord.

**Review by black-sliver — APPROVED** [review]

**Comment by black-sliver** [comment]
technically, i think "... resume operation" or something like that would be more english

**Comment by Berserker66** [comment]
Now it still says 1 hours, I just noticed. Dangit

**Comment by black-sliver** [comment]
> Now it still says 1 hours, I just noticed. Dangit

oh, i completely missed that. Maybe we should just change it to 2hrs :-D

File: WebHostLib/models.py [hunk 2]
```diff
@@ -27,7 +27,7 @@ class Room(db.Entity):
     seed = Required('Seed', index=True)
     multisave = Optional(buffer, lazy=True)
     show_spoiler = Required(int, default=0)  # 0 -> never, 1 -> after completion, -> 2 always
-    timeout = Required(int, default=lambda: 6 * 60 * 60)  # seconds since last activity to shutdown
+    timeout = Required(int, default=lambda: 1 * 60 * 60)  # seconds since last activity to shutdown
```
Comment (Berserker66) [inline WebHostLib/models.py:30]:
```suggestion
    timeout = Required(int, default=lambda: 2 * 60 * 60)  # seconds since last activity to shutdown
```

**Comment by Berserker66** [comment]
Can't be bothered to make the text dynamic, so just make it plural in content, love it XD

**Comment by strotlog** [comment]
@Berserker66 Could the number of hours be configurable somehow? Between hobbyist dev patterns and the PR feedback cycle, I find myself reusing my testing WebHost every few hours.

**Pull request merged**